spotinst_sdk2/client.py

```diff
@@ -72,9 +72,12 @@ def send_get(self, url,entity_name,query_params=None):
         else:
             self.handle_exception("getting {}".format(entity_name), result)
 
-    def send_delete(self, url, entity_name):
```

Comment:
Implement another method send_delete_with_param similar to set_put_with_param instead of modifying existing method.

Reply:
Done